edith-step/src/ReceiptInput.js

```diff
@@ -0,0 +1,92 @@
+import React from 'react';
+
+export default class ReceiptInput extends React.Component {
+  constructor(props) {
+    super(props);
+    this.state = {items: [], itemName: '', itemPrice: 0.0};
+    this.handleChange = this.handleChange.bind(this);
+    this.handleSubmit = this.handleSubmit.bind(this);
+  }
+
+  handleSubmit(e) {
+    e.preventDefault();
```

Comment:
Is this what checks that the form is set correctly? e.g. that name and price are set? if so, should we add a test for it?